app/lib/actions/candlepin/consumer/bind_entitlement.rb

```diff
@@ -0,0 +1,30 @@
+module Actions
+  module Candlepin
+    module Consumer
+      class BindEntitlement < Candlepin::Abstract
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        input_format do
+          param :organization_id, Integer
+          param :pools, Array
+        end
+
+        def plan(organization_id = nil, pools = [])
+          sequence do
+            plan_self(organization_id: organization_id, pools: pools)
+            plan_action(Katello::Organization::ManifestRefresh, Organization.find(organization_id))
```

Comment:
I don't think we need a manifest refresh if this were given an empty array of pools